cpp/sopt/credible_region.h

```diff
@@ -54,11 +53,9 @@ credible_interval(const Eigen::MatrixBase<T> &solution, const t_uint &rows, cons
                   const std::tuple<t_uint, t_uint> &grid_pixel_size,
                   const std::function<t_real(typename T::PlainObject)> &objective_function,
                   const t_real &alpha);
-}  // namespace credible_region
-}  // namespace sopt
+} // namespace sopt::credible_region
 
```

Comment:
This seems to be another place where we close a namespace and immediately open it again. Any ideas why?

Reply:
This happens in a few other files too. I think we should just merge them up (if we manage to hunt them).

But perhaps I should do that in a separate PR (with a tag of code=quality)?

Reply:
Thank you. Recorded in issue #390, which should be closed by PR #392.